src/ControllerStatusIO.c

```diff
@@ -114,23 +120,28 @@ BOOL Ros_Controller_Initialize()
     }
 
     //get the robot calibration data for multi-robot systems
-    for (i = 0; i < MAX_ROBOT_CALIBRATION_FILES; i += 1)
+    const BOOL bCalibLoadedOk = Ros_Controller_LoadGroupCalibrationData(&g_Ros_Controller);
+    //see whether the user should be notified about failures (it's OK to not
+    //have/load any calibration in some cases)
+    const BOOL bNeedToWarnAboutCalib = Ros_Controller_ShouldWarnNoCalibDataLoaded(
+        &g_Ros_Controller, g_nodeConfigSettings.publish_tf, bCalibLoadedOk);
```

Comment:
The last two arguments are swapped. 

```suggestion
        &g_Ros_Controller, bCalibLoadedOk, g_nodeConfigSettings.publish_tf);
```

Reply:
Good catch, thanks.